library_generation/new_client/new-client.py

```diff
@@ -0,0 +1,458 @@
+# Copyright 2019 Google LLC
```

Comment:
Since we have a few modifications here and there, I think it makes sense to update the copyright year